scenario12/jmeter/01-Scenario-12-ClaimMapper.jmx

```diff
@@ -2131,8 +2131,21 @@ try {
 		WebElement button = driver.findElement(By.id(&quot;loginbutton&quot;));
 		button.click();
 
-		//Returns the whole page
+		String pageSource = driver.getPageSource();
+
+		if(driver.getPageSource().contains(&quot;User Consents&quot;)) {
+
+		WebElement checkbox = driver.findElement(By.id(&quot;consent_select_all&quot;));
+		checkbox.click();
+		System.out.println(&quot;check box clicked&quot;);
+
+		WebElement approvebutton = driver.findElement(By.id(&quot;approve&quot;));
+		approvebutton.click();
+		System.out.println(&quot;approve button clicked&quot;);
+
 		return driver.getPageSource();  
+				
+		} 
```

Comment:
Better to include else statement to capture issues in loading user concent page

Reply:
Added the else statement